{{cookiecutter.project_slug}}/README.rst

```diff
@@ -1,7 +1,9 @@
+{% set is_open_source = cookiecutter.open_source_license != 'Not open source' %}
```

Comment:
I think you need to add a `-` to the end `%}` to trim the extra whitespace line that gets inserted at the top